script/lib/git.py

```diff
@@ -78,53 +78,28 @@ def get_head_commit(repo):
     return subprocess.check_output(args).strip()
 
 
-commits = {}
+def reset(repo):
+  args = ['git', 'reset']
 
-def prepare_commit(repo, author, message):
-  global commits
-  """ Commit whatever in the index is now."""
+  with scoped_cwd(repo):
+    subprocess.check_call(args)
 
-  if repo not in commits:
-    commits[repo] = []
-    
-  commits[repo].append((author, message))
 
-  return True
+def commit(repo, author, message):
+  """ Commit whatever in the index is now."""
 
-def finalize_commits():
-  global commits
-  print('\n\nfinalizing commits')
   # Let's setup committer info so git won't complain about it being missing.
   # TODO: Is there a better way to set committer's name and email?
   env = os.environ.copy()
   env['GIT_COMMITTER_NAME'] = 'Anonymous Committer'
   env['GIT_COMMITTER_EMAIL'] = 'anonymous@electronjs.org'
 
-  committed_successfully = True
-
-  for repo in commits:
-    details = commits[repo]
-    # Skip the commit when nothing to commit
-    if len(details) == 0:
-      continue
-
-    # Reset the list of commits for next time
-    commits[repo] = []
-    generated_message = 'Applied ' + str(len(details)) + ' Electron Patches\n\n'
-
-    for item in details:
-      generated_message += 'Author: ' + item[0] + '\n' + 'Message: ' + item[1] + '\n\n'
```

Comment:
Collecting all patches' descriptions into one message doesn't help, because git blame becomes unusable, so we shouldn't do it at all.